docs/GETTING_STARTED.md

```diff
@@ -134,7 +134,7 @@ does not have access to.
 
 To create the index run:
 ```
-sudo ./soci create localhost:5000/rabbitmq:latest
+sudo ./soci create --oras localhost:5000/rabbitmq:latest
```

Comment:
Should probably have a one-liner on why we're specifying `--oras`.

Reply:
Done. I rephrased the sentence above.

Reply:
I think we should be explicit about what the `--oras` flag does. Most people would read the doc before trying out the CLI, so they still won't know the reason for this flag. How about adding another line after the command: 


The `--oras` flag instructs the CLI to build an ORAS-compatible manifest, as opposed to an OCI Artifact Manifest. By default, the CLI will build the latter.

Reply:
Yes, that makes sense. Updated.